src/wiseagents/core.py

```diff
@@ -27,7 +29,7 @@ class WiseAgentCollaborationType(Enum):
 
 
 class WiseAgentTool(yaml.YAMLObject):
-    ''' A WiseAgentTool is an abstract class that represents a tool that can be used by an agent to perform a specific task.'''
+    ''' WiseAgentTool represents a tool that can be used by an agent to perform a specific task.'''
```

Comment:
I'll let @maeste comment on what was intended for this one.

Reply:
FWIW I could not see anything extending this

Reply:
I'm good with this change